docs-client/build.gradle

```diff
@@ -28,9 +31,15 @@ task buildWeb(type: NpmTask) {
 
     args = ['run', 'build']
 
-    inputs.dir('src')
-    inputs.files('package.json', 'package-lock.json')
+    inputs.property("node.version", node.version)
+    inputs.dir('src').withPathSensitivity(PathSensitivity.RELATIVE)
+    inputs.dir('node_modules').withPathSensitivity(PathSensitivity.RELATIVE)
+    inputs.files('package.json', 'package-lock.json', 'tsconfig.json', 'tsconfig-webpack.json',
+                 'webpack.config.ts').withPathSensitivity(PathSensitivity.RELATIVE)
```

Comment:
I'd suggest also naming the inputs for easier viewing in build scans. You can do that with `.withPropertyName("someName")`

Reply:
Thanks, added to the files (didn't add to the other properties since they seemed pretty straightforward)

Reply:
In general we suggest to add it to everything, otherwise you will get just `$1`, $2`, etc. in the scan, like here: https://ge.armeria.dev/c/y4by6h2mrop3u/gt25suskafqek/task-inputs?expanded=WyJzdG4yY21sa2F5c29vLSQxIiwic3RuMmNtbGtheXNvby0kMiIsInN0bjJjbWxrYXlzb28tJDMiXQ#stn2cmlkaysoo-$1